manual/cops_metrics.md

```diff
@@ -38,7 +38,7 @@ Attribute | Value
 --- | ---
 CountComments | false
 Max | 25
-ExcludedMethods | 
+ExcludedMethods |
 
```

Comment:
Probably you should also change whatever is adding 2 blank lines here instead of one.